massgov/mayflower-patternlab
# DP-3527 / DP-3528 obsolete patterns

## Status
- on hold until the 5.8 release is completed

## Description
Archiving or removing unused patterns.

## Related Issue / Ticket
- https://jira.state.ma.us/browse/DP-3527
- https://jira.state.ma.us/browse/DP-3528

## Steps to Test
- In your local Pattern Lab check the archive folders contain the patterns in DP-3528
- In your local Pattern Lab check if patterns in DP-3528 are no longer there

This PR includes changes from https://github.com/massgov/mayflower/pull/571
